src/Resources/skeleton/registration/RegistrationController.tpl.php

```diff
@@ -7,17 +7,36 @@
 <?php if ($authenticator_full_class_name): ?>
 use <?= $authenticator_full_class_name; ?>;
 <?php endif; ?>
+<?php if ($will_verify_email): ?>
+use <?= $verify_email_security_service; ?>;
+use Symfony\Bridge\Twig\Mime\TemplatedEmail;
+<?php endif; ?>
 use Symfony\Bundle\FrameworkBundle\Controller\<?= $parent_class_name; ?>;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
+<?php if ($will_verify_email): ?>
+use Symfony\Component\Mime\Address;
+<?php endif; ?>
 use Symfony\Component\Routing\Annotation\Route;
 use Symfony\Component\Security\Core\Encoder\UserPasswordEncoderInterface;
 <?php if ($authenticator_full_class_name): ?>
 use Symfony\Component\Security\Guard\GuardAuthenticatorHandler;
 <?php endif; ?>
+<?php if ($will_verify_email): ?>
+use SymfonyCasts\Bundle\VerifyEmail\Exception\VerifyEmailExceptionInterface;
+<?php endif; ?>
 
 class <?= $class_name; ?> extends <?= $parent_class_name; ?><?= "\n" ?>
 {
+<?php if ($will_verify_email): ?>
+    private $verifier;
+
+    public function __construct(VerifyEmail $verifier)
```

Comment:
It occurred to me that `verifier` was a little bit generic of a name - a more specific name would help me remember what this is. So I was thinking `$emailVerifier` (here and on the property). Then I though: should the class also be called `EmailVerifier`? That makes it sound more like a service and less like a model.

Reply:
done